inventory/sample.libvirt.example.com.d/inventory/hosts

```diff
@@ -0,0 +1,44 @@
+[infra]
+cloud-host.local
+
+[infra_vms]
```

Comment:
How are users of this playbook supposed to define how much CPU and RAM each VM will be allocated?